docs-content/getting-started/3_client-sdk/2_nextjs.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Next.js
+title: Next.js Quick-start
```

Comment:
```suggestion
title: Next.js Quick Start
```

Reply:
These `Quick-start` to `Quick Start` edits are optional, but I wanted to make sure that the other comment was addressed, hence the "requested changes".